draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -717,6 +774,175 @@ Steps:
 23. Output (x, y)
 ~~~
 
+### Boneh-Franklin Method {#supersingular}
+
+The map2curve_bf(alpha) implements the Boneh-Franklin method {{BF01}} which
+covers the case of supersingular curves `E: y^2=x^3+B`.
+
+**Preconditions**
+
+This algorithm works for any Weierstrass curve over `F_q` such that `A=0` and
```

Comment:
It also requires that `b=1`, no? Can you explain why it works for other values of `b`?

Reply:
It works for any `B` since there exist always a cubic root of any element in `Fp`.